### ISSUE TITLE
__init__.py:6-8: After #37 is completed add execution of...

### ISSUE THREAD

**Issue by 0pdd** [timelesslounge/timelessis#98]
The puzzle `50-137f7115` from #50 has to be resolved: 

https://github.com/timelesslounge/timelessis/blob/7b5bc2913d76521b6a81152acdcf0618c74235b5/__init__.py#L6-L8

The puzzle was created by Krzysztof Krason on 02-Feb-19. 

  

If you have any technical questions, don't ask me, submit new tickets instead. The task will be \"done\" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@bkuzmic[/z](https://www.0crat.com/u/bkuzmic) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0pdd** [comment]
The puzzle `50-137f7115` has disappeared from the source code, that's why I closed this issue.

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @bkuzmic[/z](https://www.0crat.com/u/bkuzmic), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#98](https://github.com/timelesslounge/timelessis/issues/98) is now out of scope

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @bkuzmic[/z](https://www.0crat.com/u/bkuzmic)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)
